gel/_internal/_reflection/_callables.py

```diff
@@ -71,6 +71,10 @@ def key(self) -> CallableParamKey:
         else:
             return self.index
 
+    @functools.cached_property
+    def sort_key(self) -> str:
+        return str(self.key)
```

Comment:
when in CI you can also mix hashseed right in

Reply:
What do you mean?  This sort key should be stable regardless of seed.